superdesk/default_settings.py

```diff
@@ -94,11 +94,6 @@ def local_to_utc_hour(hour):
 #:
 ELASTIC_DEFAULT_SIZE = 10
 
-#: allow wildcards in query string
-#:
-#: https://discuss.elastic.co/t/configuring-the-standard-tokenizer/8691/5
-ELASTIC_QUERY_STRING_ANALYZE_WILDCARD = False
```

Comment:
This will still be required, as it's used by search to fill in from a client search request (and not a creation of index & settings)

Reply:
Fixed.